src/Common/src/Interop/Interop.Odbc.cs

```diff
@@ -94,19 +94,19 @@ internal static partial class Odbc
         //            SQLSMALLINT *StringLength, SQLPOINTER NumericAttribute);
         // #endif
 
-        [DllImport(Interop.Libraries.Odbc32)]
+        [DllImport(Interop.Libraries.Odbc32, CharSet = CharSet.Unicode)]
         internal static extern /*SQLRETURN*/ODBC32.RetCode SQLColumnsW(
             /*SQLHSTMT*/OdbcStatementHandle StatementHandle,
-            [In, MarshalAs(UnmanagedType.LPWStr)]
```

Comment:
`MarshalAs(UnmanagedType.LPWStr)`

Why is this not needed any more?

Reply:
https://github.com/dotnet/coreclr/pull/23664#issuecomment-480773357 - the marshallng as Unicode should make this UTF-16 (based on [this patch](https://github.com/filipnavara/corefx/commit/ed232072f636f812426daf5c21e30e0f377c0491) by Filip)

Reply:
`LPWStr` is the default marshaling for `string` when you have `CharSet.Unicode` explicitly set. The default `CharSet` is `CharSet.Ansi` for legacy compatibility reasons.  Note that `[In]` isn't needed either.

Reply:
I did leave the In attribute there because I didn't want to risk changing any semantics other than keeping strings consistent.

Reply:
Understood. :) It doesn't change the semantics. The string will be pinned with or without the `[In]`.

Reply:
Where would you like the In attributes removed; just that line, function, or all instances on strings?

Reply:
It doesn't need to be on any of the string parameters. It isn't a huge deal, it's just nice to get in line with our [guidelines](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/interop-guidelines.md) since we're changing this code anyway. Hopefully it will reduce the "is this special?" confusion. :)